src/_main_/settings.py

```diff
@@ -39,8 +39,8 @@
     RUN_CELERY_LOCALLY = True
 
 # Database selection, development DB unless one of these chosen
-IS_PROD = True
-IS_CANARY = False
+IS_PROD = False
+IS_CANARY = True
```

Comment:
nope.

Reply:
This code is from the canary branch. I will change it after merge